safe-global/safe-homepage
# Add specific category filters to ecosystem page

## What it solves

Part of #176

## Other changes

- Adds header background particle image
- Adds project links

## How to test

1. Open `/ecosystem`
2. Click on the links below the search bar
3. Observe the results are filtered by those categories

## Screenshots

<img width="1053" alt="Screenshot 2023-05-09 at 16 41 36" src="https://github.com/safe-global/safe-homepage/assets/5880855/7e2517b3-0e6c-4e9c-b625-c19ba74387b5">
<img width="1476" alt="Screenshot 2023-05-09 at 16 48 06" src="https://github.com/safe-global/safe-homepage/assets/5880855/c3d3a78e-7cff-43fe-be19-adf59828fd03">
